WordPress/Classes/ViewRelated/Post/Scheduling/CalendarMonthView.swift

```diff
@@ -73,9 +72,23 @@ class CalendarMonthView: UIView {
         ])
 
         addSubview(stackView)
+
+        stackView.addConstraints([
+            collectionView.heightAnchor.constraint(equalToConstant: 240),
+            collectionView.widthAnchor.constraint(equalToConstant: 375)
```

Comment:
We try not to just have hard-coded values. We tend to use either a `struct` or `enum` to give constants indicative names. If you look for examples of `struct Constants` or `enum Constants`, that should give you an idea.